doc/source_tech_preview/deployment/configure.rst

```diff
@@ -118,7 +118,7 @@ The Ceph admin keyring can be obtained by running the following on ceph host.
 
 .. code-block:: yaml
 
-  sudo ceph auth get-key client.admin | base64
+  echo $( sudo ceph auth get-key client.admin ) | base64
```

Comment:
I think this could be further improved by using `base64 <(sudo ceph auth get-key client.admin)`

Reply:
could you explain advantage over "echo"?

Reply:
adds a \n to string that is getting base64 encoded.  had issues authenticating with ses without /n.

Reply:
sorry the question was for @jesusaurus.

Reply:
It's not a very important difference, but it spawns fewer processes/subshells by giving the base64 process a direct file handle for the ceph output.